### DISCUSSION TITLE
How to implement SimRank in pytorch geometric

### DISCUSSION THREAD

**Discussion by OswaldoBornemann** [pyg-team/pytorch_geometric#5043] [General]
How to implement SimRank in pytorch geometric ?

**Comment by rusty1s** [comment]
Can you specify your question and how you want to use PyG for SimRank computation? PyG is a library for simplifying the construction of Graph Neural Networks - it is not a general purpose library for graph algorithms. You could have a look at https://networkx.org/documentation/stable/reference/algorithms/generated/networkx.algorithms.similarity.simrank_similarity.html instead.

**Reply by OswaldoBornemann** [reply]
Thanks @rusty1s
